gitlab.go

```diff
@@ -117,6 +117,7 @@ type Client struct {
 	Epics                 *EpicsService
 	Events                *EventsService
 	Features              *FeaturesService
+	FreezePeriod          *FreezePeriodsService
```

Comment:
In order for this to work as expected, you also need to add this to line ~284:

```go
c.FreezePeriod = &FreezePeriodsService{client: c}
```

Reply:
I've updated the file with the correct naming and added the requested line.